content/automate/ManageIQ/Transformation/TransformationHosts/StateMachines/TransformationHostRole.class/check.yaml

```diff
@@ -0,0 +1,15 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: Check
+    inherits: 
+    description: 
+  fields:
+  - State1:
+      value: "/Transformation/TransformationHosts/${#transformation_host_type}/RoleCheck"
+      on_error: "/Transformation/TransformationHosts/${#transformation_host_type}.MarkAsDisabled"
```

Comment:
Where is ```MarkAsDisabled``` defined?

Reply:
There is a TagAsEnabled? Does this needs to be renamed.

Reply:
You're right. It is TagAsDisabled. I have renamed it.